src/swarm-lang/Swarm/Language/JSON.hs

```diff
@@ -32,8 +32,38 @@ instance ToJSON Syntax
 instance ToJSON Value where
   toJSON = genericToJSON optionsMinimize
 
-instance FromJSON Value where
-  parseJSON = genericParseJSON optionsMinimize
+-- instance FromJSONE (CtxMap CtxTree t) Value where
+--   parseJSONE = withObjectE "Value" $ \v -> case Ae.toList v of
```

Comment:
Could you please add a TODO comment with a linked Issue?

Reply:
Good idea, done!